setup.py

```diff
@@ -31,12 +31,12 @@
     "Pillow>=4.0.0,<8.0.0",
     "beautifulsoup4>=4.8,<4.9",
     "html5lib>=0.999,<2",
-    "Unidecode>=0.04.14,<2.0",
```

Comment:
I'm not sure if this was correct or if I should leave unidecode in the requirements till a future release can remove it?

It doesn't appear to be required but rather it now an optional dependency you could install to use AbstractFormField `_migrate_legacy_clean_name`.